src/qtcore/qml/elements/QtQuick/MouseArea.js

```diff
@@ -81,6 +83,16 @@ registerQmlType({
         self.containsMouse = false;
         self.exited();
     }
+    this.dom.onmouseover = function(e) {
+        self.containsMouse = true;
+        self.dom.style.cursor = cursorShapeToCSS();
+        self.entered();
+    }
+    this.dom.onmouseout = function(e) {
+        self.containsMouse = false;
+        self.dom.style.cursor = "auto";
+        self.exited();
+    }
```

Comment:
I don't understand why we have to do this. If you simply watch for the `cursorShape` property, and set the `dom.style.cursor` value whenever it changes, won't it do the same thing ? Do we really need to update the css attribute everytime the mouse enters or exits it ?